src/data/faq.json

```diff
@@ -352,19 +352,54 @@
+                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?",
+                        "anchor": "admission_policy_2gplus_not_shown",
+                        "active": false,
+                        "textblock": [
+                            "The 2G+ status proof results, among other things, from the combination of vaccination, recovery <b>and</b> test certificates. If the data within the certificates, for example name or date of birth, do not match, the Corona-Warn-App cannot match these certificates and determine the corresponding status. In this case, the Corona-Warn-App will show you two certificates in the overview with 3G and 2G rules assigned to them respectively. Therefore, make sure that your name is entered correctly and fully when issuing the certificate, and contact the issuing office of the certificate in case of errors in the entry of the name or date of birth.",
+                            "A test certificate officially confirms a negative test result in the form of a QR code in the Corona-Warn-App. You can request the official test certificate for PCR and rapid tests when you register a test in the Corona-Warn-App. To do this, you can open the universal QR code scanner in the bottom center of the CWA and scan the test registration QR code. You will receive this either while booking the test or on-site during testing, provided the provider is connected to the Corona-Warn-App. After scanning the QR code, a window will automatically open where you can request the test certificate. You can find more information in this blog: <a href='../blog/2021-06-24-cwa-version-2-4' target='_blank'>Project team releases rapid test partner search and Corona-Warn-App version 2.4 with digital test certificate</a>",

```

Comment:
"provided the provider" => 
"if the provider"

(otherwise it sounds a bit clumsy)